components/icon/stories/icon.stories.js

```diff
@@ -79,12 +59,22 @@ export default {
 			},
 			control: "color",
 		},
-		useRef: { table: { disable: true } },
+		useRef: {
+			name: "Use sprite sheet reference",
+			description: "Storybook only: whether to display an SVG with a `<use>` reference to the icon within a loaded sprite sheet. This improves Storybook performance, especially for multiple icons. When set to `false`, the icon file's full markup is used.",
+			type: { name: "boolean" },
+			table: {
+				type: { summary: "boolean" },
+				category: "Advanced",
+			},
+			control: "boolean",
+		},
 	},
 	args: {
 		rootClass: "spectrum-Icon",
 		setName: "workflow",
-		iconName: "ABC",
+		iconName: "Color",
+		uiIconName: "Checkmark400",
 		size: "xl",
 		useRef: true,
 	},
```

Comment:
Let's add the Figma links here too: https://www.figma.com/design/9qeVZSJ9t0kv6r7njzgHx7/S2-%2F-Styles-visualizer-(WIP)?node-id=295-24257&t=ZC7fyaQ0VQYQ5VYM-1

Reply:
I've added this Figma design link to the parameters.